packages/x-license/package.json

```diff
@@ -18,6 +18,7 @@
     "dist/*"
   ],
   "license": "See LICENSE file",
+  "sideEffects": false,
```

Comment:
This one is not as it affects the global scope?

Reply:
True, but it's not the type of side effect we care here. A side effect that is relevant for tree-shaking would be a global polyfill: https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free.